src/tests/StreamerTest.cpp

```diff
@@ -129,9 +158,12 @@ TEST_F(StreamerProcessTest, EndOfPartition) {
 }
 
 TEST_F(StreamerProcessTest, PollingError) {
-  StreamerStandIn TestStreamer;
+  StreamerOptions Options;
```

Comment:
These four lines appear in every StreamerProcessTest test. Could they be put in the SetUp to reduce duplication?